app/build.gradle

```diff
@@ -33,15 +35,21 @@ repositories {
 }
 
 dependencies {
-  def inAppPaymentsSdkVersion = '1.2.0'
-  implementation "com.squareup.sdk.in-app-payments:card-entry:$inAppPaymentsSdkVersion"
-  implementation "com.squareup.sdk.in-app-payments:google-pay:$inAppPaymentsSdkVersion"
-  implementation 'com.squareup.retrofit2:retrofit:2.5.0'
-  implementation 'com.squareup.retrofit2:converter-moshi:2.5.0'
-  def supportLibraryVersion = '28.0.0'
-  implementation "com.android.support:appcompat-v7:$supportLibraryVersion"
-  implementation "com.android.support:design:$supportLibraryVersion"
-  implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+  implementation "androidx.appcompat:appcompat:1.1.0"
```

Comment:
nit: add a `def androidxVersion = '1.2.1`?

Reply:
I just pulled from what's internally used, so we _could_ jump to consistent-version (which I guess is 1.1.3) or most-recent, but I opted for most-tested.